src/main/cluster-detectors/distribution-detector.ts

```diff
@@ -83,6 +87,10 @@ export class DistributionDetector extends BaseClusterDetector {
     return this.cluster.contextName.startsWith("microk8s");
   }
 
+  protected isKind() {
+    return this.cluster.contextName.startsWith("kind-");
```

Comment:
Why don't we use the cluster name that is associated with the context? Users can easily make multiple contexts (call one dev, one prod) but the cluster name is less likely to be changed(?)

Reply:
I thought kind prefixes context names always with `kind-`, even if user gives a custom name for a cluster.

Reply:
Will verify this asap ^

Reply:
Tested, it starts always with `kubernetes-admin@kind-` ... so there was a bug (fixed now).

Reply:
> I thought kind prefixes context names always with kind-, even if user gives a custom name for a cluster.

It does. But I can add my own contexts that use the kind cluster:
```
kubectl config set-context prod-frontend --cluster=kind-cluster --user=kind-cluster
```

Or is this not the same context? Am I out of context?